src/v2/api/services/RoleService.ts

```diff
@@ -103,4 +107,77 @@ export class RoleService {
 
     await this.checkRole(userRoles, parent);
   }
-}
\ No newline at end of file
+
+
+  async transferRole (captainUserId : string, studentId: string) {
```

Comment:
you can simplify name by removing `User`, because captain is already user, you don't need to specify it in your name of variable